roxie/ccd/ccdmain.cpp

```diff
@@ -1520,22 +1520,17 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                         Owned<const ISyncedPropertyTree> tlsConfig;
                         if (serviceTLS)
                         {
-                            protocol = "ssl";
 #ifdef _USE_OPENSSL
-                            if (isContainerized())
+                            protocol = "ssl";
```

Comment:
This roxie config change looks good... but may want to fix line wrapping in commit message?